src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public final String value;
+    protected boolean isPrivate;
+	
+    /**
+     * Construct new contact object
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)